docs/concepts/11-versions.md

```diff
@@ -0,0 +1,609 @@
+
+```
+1.2.0-dev.20060102T150405Z    # Development release for 1.2.0
+1.2.0                        # Final release
+```
+
+### Key Properties
+
+- **Ordering**: Development releases are sorted **before** their corresponding release version
+- **Testing**: Perfect for staging environments and pre-production testing
+- **Timestamps**: Multiple development releases are ordered by timestamp (higher = newer)
+
+### Rapid Infrastructure Testing
+
+Development releases are a powerful means of rapidly testing Infrastructure as Code (IaC) changes against real cloud resources:
+
+**Local Development + Real Infrastructure:**
+- Develop bundle changes locally on your machine
+- Publish development releases without affecting production bundles
+- Deploy development releases to real cloud environments for comprehensive testing
+
+**Opt-in Testing Network:**
+- Set up multiple packages using release channels with "Enable Development Releases" toggle
+- These packages automatically pick up your latest development releases and run the full deploy/apply pipeline
+- Run Terraform plans and compliance scans against actual provisioned resources
+- Validate infrastructure changes with real cloud provider APIs
+
+**Practical Workflow:**
+```bash
+# 1. Make IaC changes locally
+vim src/main.tf
+
+# 2. Publish development release for testing
+mass bundle publish --development-release
+
+# 3. Visit the output URL to monitor all test infrastructure
+# 4. Watch plan/apply/compliance pipelines execute automatically
+# 5. Review Terraform plans and compliance results from real infrastructure
+# 6. Iterate with new development releases or publish final release
+```
+
+This approach lets you stand up multiple real-world examples of your infrastructure changes before committing to a final release, ensuring your bundles work correctly across different cloud environments and configurations.
+
+## Release Channels
+
+Release channels use tilde (`~`) constraints to specify compatible version ranges. When a package uses a release channel, it **automatically runs the full deploy/apply pipeline** whenever a new version matching that channel is published. This includes compliance checks, Terraform plans, and all other validation steps. This automates infrastructure version management.
+
+### Major Channel (`~1`)
+Matches the latest version within major version 1:
+- `~1` could resolve to `1.5.3` (latest in major 1)
+- Will **not** upgrade to `2.0.0`
+- Automatically deploys when new minor or patch versions are published (e.g., `1.6.0`, `1.5.4`)
+
+### Minor Channel (`~1.1`)  
+Matches the latest patch within minor version 1.1:
+- `~1.1` could resolve to `1.1.7` (latest patch in 1.1.x)
+- Will **not** upgrade to `1.2.0`
+- Automatically deploys when new patch versions are published (e.g., `1.1.8`)
+
+### Latest Channel (`latest`)
+Matches the newest stable release:
+- `latest` could resolve to `2.3.1` (newest stable release)
```

Comment:
No context for why it would resolve this specific version (no example given, just an arbitrary version).